### ISSUE TITLE
Error: undefined method `path' for Hbc::Caskroom:Module

### ISSUE THREAD

**Issue by y0y0z** [buo/homebrew-cask-upgrade#97]
i am running `brew cu -y -a` daily. since today, i got the following error
```bash
$ brew cu -y -a
Error: undefined method `path' for Hbc::Caskroom:Module
/usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/lib/extend/hbc.rb:1:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/lib/bcu.rb:6:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/cmd/brew-cu.rb:32:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Homebrew/utils.rb:18:in `require?'
/usr/local/Homebrew/Library/Homebrew/brew.rb:101:in `<main>'
```

clearly it has something to do with [this commit](https://github.com/buo/homebrew-cask-upgrade/commit/8fb37d33541d3aa015a9c8b60afd97ec634e9c49).

when i revert the commit, everything works as expected.
any idea?

**Comment by SkyRaker** [comment]
I met the same issue TAT.

**Comment by randpop** [comment]
Same here...

**Comment by ondrejfuhrer** [comment]
Hey @y0y0z , try running `brew update` before, seems like you have old version of brew running with the new version of `brew-cask-upgrade`

**Comment by zym013579** [comment]
@ondrejfuhrer In my opinion, seems like he have old version of `brew-cask-upgrade` running with the new version of brew

**Comment by raphael-pix4d** [comment]
I have the same

**Comment by FateLight** [comment]
me too

**Comment by alioguz** [comment]
Try running `cd "$(brew --repo)" && git fetch && git reset --hard origin/master && brew update`

**Comment by ondrejfuhrer** [comment]
@zym013579 Nope, since the `Hbc::Caskroom.path` is the new way of getting the caskroom path 🙂 

@alioguz Was just writing the same thing 👍

**Comment by SkyRaker** [comment]
@alioguz Cooooool!

**Comment by pavstor** [comment]
Had the same problem. Solution by @alioguz fixed it (thanks!)

**Comment by buo** [comment]
I fixed this issue in the commit 36b26b3f2a2c6fe295b6a507d1d73ab28bc7f6ad by trying to call both `Hbc.caskroom` and `Hbc::Caskroom.path`. Thanks!
